src/ui/components/Library/Team/View/NewTestRuns/Overview/RunResults.tsx

```diff
@@ -78,7 +78,7 @@ function TestStatusGroup({
 }) {
   const [expanded, setExpanded] = useState(true);
 
-  const tree = useFileNameTree(testGroup, filterByText);
+  const tree = useFileNameTree(testGroup, filterByText, true);
```

Comment:
I don't think we need to make this an option. This is the only usage (right?) so it can just be the behavior that we search both the test titles and the file name.

Reply:
In old view, do we still want to search just the file name? Because that's what we highlight there.

Reply:
The old view isn't relevant in this branch. It will be replaced when to land this code in main